bus_analyzer/bus_analyzer/main.py

```diff
@@ -220,7 +221,8 @@ def analyze_hour(db_manager, prototypes, existing_segments, base_dir, channel, q
 
 	return [segment[1] for segment in segments_to_do]
 
-def post_process(db_manager, segments):
+@timed(normalize=lambda result, db_manager, segments: len(segments))
```

Comment:
since you're adding `channel` as an arg to the function it also needs to be in this lambda (it gets the same args as the function)
```suggestion
@timed(normalize=lambda result, db_manager, segments, channel: len(segments))
```